src/LabelSelector/LabelChip/LabelChip.tsx

```diff
@@ -12,31 +13,36 @@ export default function LabelChip({
   size = "medium",
   variant = "filled",
   visible = true,
+  selected = false,
+  description = "",
   ...props
 }: LabelChipProps) {
   // return the styled chip component
   return (
-    <Chip
-      {...props}
-      className="label-chip"
-      clickable={clickable}
-      sx={{
-        "&:hover": {
-          backgroundColor: clickable ? darken(color, 0.2) : color
-        },
-        backgroundColor: color,
-        color: theme => theme.palette.getContrastText(color),
-        [`& .${chipClasses.deleteIcon}`]: {
-          color: theme => alpha(theme.palette.getContrastText(color), 0.7)
-        },
-        [`& .${chipClasses.deleteIcon}:hover`]: {
-          color: theme => theme.palette.getContrastText(color)
-        },
-        visibility: visible ? "visible" : "hidden"
-      }}
-      label={<NoWrapTypography variant="inherit">{label}</NoWrapTypography>}
-      size={size}
-      variant={variant}
-    />
+    <Tooltip title={description}>
```

Comment:
![image](https://github.com/user-attachments/assets/b5d6a65d-1173-4740-9b8a-4c62e3d85e8e)
<img width="346" alt="image" src="https://github.com/user-attachments/assets/35f478cc-cbb9-4fd2-9289-4df0fc431b79">

The Figma shows that the tooltips should be left aligned to the LabelChip (at least in the Label Selection popover), but they appear centered in Storybook. @Sowbhagya-ipg can you confirm the expectation here?

Reply:
Left align it. @m4manjesh